plugins/experimental/remap_stats/remap_stats.cc

```diff
@@ -44,11 +44,13 @@ struct config_t {
 // From "core".... sigh, but we need it for now at least.
 extern int max_records_entries;
 
-static void
+namespace
```

Comment:
Oooh, moving up to eleventy style! Radical.

Reply:
i can restore it back to static